test/orm/test_query.py

```diff
@@ -96,32 +96,38 @@ def test_with_session(self):
 class OnlyReturnTuplesTest(QueryTest):
     def test_single_entity_false(self):
         User = self.classes.User
-        row = create_session().query(User).only_return_tuples(False).first()
+        query = create_session().query(User).only_return_tuples(False)
+        assert query.is_single_entity is True
```

Comment:
can you use is_true(query.is_single_entity) for these

from sqlalchemy.testing import is_true